README.md

```diff
@@ -12,6 +12,9 @@ Community (college) maintained list of Algorithms and Data Structures implementa
 
 | Algorithm | C | CPP | Java | Python | Golang |
 |:--------------|:----------------:|:----------------:|:----------------:|:-----------------:|:-----------------:|
+
+| [Bin Sort](http://www.cdn.geeksforgeeks.org/bucket-sort-2/)| | | [\[X\]](bin_sort/BinSort.java)| | |
```

Comment:
Extra diff added. Maybe because of rebase. Don't worry, just do a commit removing these 2 lines.